ocs_ci/ocs/node.py

```diff
@@ -1073,3 +1073,64 @@ def generate_node_names_for_vsphere(count, prefix="compute-"):
         f"{prefix}{current_compute_suffix + node_count}"
         for node_count in range(1, count + 1)
     ]
+
+
+def check_taint_on_nodes(taint=None):
+    """
+    Function to check for particular taint on nodes
+
+    Args:
+        taint (str): The taint to check on nodes
+
+    """
+    taint = taint if taint else "node.ocs.openshift.io/storage=true:NoSchedule"
```

Comment:
consider setting the `taint` function arg with a default value of `constants.OPERATOR_NODE_TAINT`.
Then you will not need this line